server/src/test/java/org/opensearch/search/aggregations/bucket/terms/SignificantTextAggregatorTests.java

```diff
@@ -104,7 +105,8 @@ protected List<String> unsupportedMappedFieldTypes() {
         return Arrays.asList(
             BinaryFieldMapper.CONTENT_TYPE, // binary fields are not supported because they do not have analyzers
             GeoPointFieldMapper.CONTENT_TYPE, // geopoint fields cannot use term queries
-            FlatObjectFieldMapper.CONTENT_TYPE // flat_object fields are not supported aggregations
+            FlatObjectFieldMapper.CONTENT_TYPE, // flat_object fields are not supported aggregations
+            ConstantKeywordFieldMapper.CONTENT_TYPE // binary fields are not supported because they do not have analyzers
```

Comment:
If we add the `ValueFetcher`, does the field work with aggregations? (If not, we should update the comment to explain why we can't support aggregations.)

Reply:
@msfroh 

[AggregatorTestCase.testSupportedFieldTypes](https://github.com/opensearch-project/OpenSearch/blob/main/test/framework/src/main/java/org/opensearch/search/aggregations/AggregatorTestCase.java#L741) Test failure for the  SignificantTextAggregatorTests Implementation was because  SignificantTextAggregatorTests requires the field to support analyzer

On the other hand other Aggregator Implementations were fixed by adding the default value
https://github.com/opensearch-project/OpenSearch/pull/12285/files#diff-ab111b9143dab7a4d3b9c3150885ff61de6f1815e98e06e398a9745bfefc20d2R770